front/lib/api/assistant/messages.ts

```diff
@@ -517,7 +610,8 @@ export async function batchRenderMessages<V extends RenderMessageVariant>(
   auth: Authenticator,
   conversationId: string,
   messages: Message[],
-  viewType: V
+  viewType: V,
+  { withContentParsing }: { withContentParsing?: boolean } = {}
```

Comment:
nit: `withParsedContent` would kind of make more sense as a name to me.

But instead, should we just have `parsedContents` be part of the agent message type ?

Reply:
i made parsedContents a part of AgentMessageType
do we always want to return it ? if so then yes withParsedContent makes no sense
i stated that we don't always include it

Reply:
I think we can remove withParsedContent


Any time we return `contents` we should also return `parsedContents` since that isn't doing any network call it's essentially free.